### PR TITLE
Updating sanity test with additional cases

### PR DESCRIPTION
**Pull request by abhinandanpb**
Added 2 cases : 
1) Continuous addition and deletion of Containers created and testing the ping
2) Continuous addition and deletion of Configs.

### CODE REVIEW

File: systemtests/twohosts/sanity_test.go [hunk 1]
```diff
@@ -1352,3 +1352,195 @@ func TestTwoHostsVxlanAddDelNetworkConsul_sanity(t *testing.T) {
...
+		utils.DockerCleanup(t, node2, "myContainer2")
+	}()
+
+	ipAddress = utils.GetIPAddress(t, node1, "purple-myContainer3", u.EtcdNameStr)
+	utils.StartClient(t, node2, "myContainer4", ipAddress)
+	defer func() {
+		utils.DockerCleanup(t, node2, "myContainer4")
+	}()
+	for i := 0; i < 5; i++ {
+		jsonCfg =
```
Comment (jainvipin) [inline systemtests/twohosts/sanity_test.go]: This configuration should only consist of incremental change i.e. only container related ep add/delete, not the entire desired configuration

File: systemtests/twohosts/sanity_test.go [hunk 2]
```diff
@@ -1352,3 +1352,195 @@ func TestTwoHostsVxlanAddDelNetworkConsul_sanity(t *testing.T) {
...
+                    "Host"              : "host2"
+                },
+                {
+                    "Container"         : "myContainer4",
+                    "Host"              : "host2"
+                } ]
+            } ]
+        } ]
+        }`
+		utils.ApplyDesiredConfig(t, jsonCfg, testbed.GetNodes()[0])
```
Comment (jainvipin) [inline systemtests/twohosts/sanity_test.go]: We don't want to specify the desiredConfig, please use add configuraiton here.

**Comment by jainvipin** [comment]
@abhinandanpb - LGTM
Not sure why sanity would fail with your changes...

**Comment by erikh** [comment]
Please squash your commits with `git rebase -i`. If you're having trouble, reach out to me, or I can do it for you.

**Pull request closed without merging**